README.md

```diff
@@ -2,10 +2,10 @@
 Helpers for building [single-spa](https://github.com/CanopyTax/single-spa) applications which use Angular Cli.
 
 ## Example
-An example can be found in the [single-spa-examples](https://github.com/PlaceMe-SAS/single-spa-angular-cli-examples) repository.
+An example can be found in the [single-spa-angular-cli-examples](https://github.com/PlaceMe-SAS/single-spa-angular-cli-examples) repository.
```

Comment:
I didn't know this examples repo existed until now, but it's awesome! Would you mind if I added a link to it at https://github.com/CanopyTax/single-spa/tree/master/examples? Also, I'd like to update https://github.com/CanopyTax/single-spa/blob/master/docs/single-spa-ecosystem.md#help-for-frameworks to point to single-spa-angular-cli for angular (in addition to single-spa-angular2, and even recommend single-spa-angular-cli over single-spa-angular-2)

Reply:
Yes perfect!

Reply:
Updated the docs with https://github.com/CanopyTax/single-spa/commit/1dbafc1eda0f36e2954f687a760af56c3dfbeaca and https://github.com/CanopyTax/single-spa-angular2/commit/8e3accb2d5949ef119e8e93240e1778ce60fc65d